src/runtime_src/core/common/device.cpp

```diff
@@ -92,8 +92,7 @@ get_xclbin_uuid() const
   catch (const query::no_such_key&) {
   }
 
-  // Emulation mode likely, just return m_xclbin_uuid which reflects
-  // the uuid of the xclbin loaded by this process.
+  // Emulation mode likely
```

Comment:
Is this reverting a comment that was recently added.   I'd like to preserve the comment.

Reply:
Restored the comment to the same as master in the latest commits.